docs/03-extensions/payzen/index.mdx

```diff
@@ -1,16 +1,26 @@
 ---
 title: "Payzen"
-description:
-  "The Payzen extension allows customer to pay using Payzen."
+description: "The Payzen extension allows customer to pay using Payzen."
 ---
 
 <p>{frontMatter.description}</p>
 
-The implementation use [PayZen embedded form / Javascript with REST API](https://payzen.io/fr-FR/rest/V4.0/javascript/) to create payments.
+The implementation use
+[PayZen embedded form / Javascript with REST API](https://payzen.io/fr-FR/rest/V4.0/javascript/)
+to create payments.
 
 :::danger
 
-This extension is only compatible with Magento1/OpenMage, we are working on Magento2 compatibility before 3.4.0.
+This extension is only compatible with Magento1/OpenMage, we are working on
+Magento2 compatibility before 3.6.0.
```

Comment:
**question:** Should we specify in which version it will be available, given the current "roadmap"?

Reply:
It should be compatible with 3.6.0 (code is nearly finished)